README.md

```diff
@@ -1,12 +1,66 @@
-# Accessible diff viewer
+---
 
-The repository contains a work in progress diff viewer with an emphasis on being accessible.
+# Diffcessible: An accessible diff viewer
 
-If you want to try it out:
+Diffcessible is an terminal based diff viewer with an emphasis on being accessible. Our goal is to build a diff viewer on top of Git diff that is accessible to screen readers.
+
+## Overview
+
+Diffcessible currently has two main components:
```

Comment:
This section is nicely written but I think it will get out of date pretty quickly as the project evolves.
I think it should be shorter with just a short description for each item.

Also, I think a non-numbered list would be better here.

Reply:
Resolved.